packages/shared/src/components/widgets/ToggleWeekStart.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { isNullOrUndefined } from '../../lib/func';
+import { DayOfWeek } from '../../lib/date';
+import { Radio, ClassName as RadioClassName } from '../fields/Radio';
+import useProfileForm from '../../hooks/useProfileForm';
+
+const getDefaultStartOfWeek = (weekStart?: number): string => {
+  if (isNullOrUndefined(weekStart)) {
+    return DayOfWeek.Monday.toString();
+  }
+
+  return (weekStart as number).toString();
+};
+
+export const ToggleWeekStart = ({
+  className,
+}: {
+  className?: RadioClassName;
+}): ReactElement => {
+  const { user } = useAuthContext();
+  const { updateUserProfile } = useProfileForm();
+
+  const toggleWeekStart = (value: string) => {
+    updateUserProfile({ weekStart: parseInt(value, 10) });
```

Comment:
:aaaaaa: Not your fault, but super annoying that certain things update in realtime and some don't 😂

Reply:
😬